draft-ietf-jsonpath-base.md

```diff
@@ -1300,8 +1300,9 @@ the comparison is between values which are both numbers or both strings and whic
       lower Unicode scalar value than the second string or if both strings start with the same Unicode scalar value and
       the remainder of the first string compares less than the remainder of the second string.
 
-Note that comparisons using the operator `<` yield false if either value being
-compared is an object, array, boolean, or `null`.
+Note that comparisons using the operator `<` yield false if either side of the
```

Comment:
Independent of your change:  I don't think this is a note.

```suggestion
Comparisons using the operator `<` yield false if either side of the
```

Reply:
@cabo The reason I thought it should be a note is that the behaviour of `<` is already fully specified and this is just a summary. With that in mind, is it still better to remove the note?

Reply:
That's what I get for looking at a change in isolation.  You are right, and with the additional complexity necessary, it is indeed questionable whether this note is needed.  I have a mild preference for it to stay, but wouldn't mind at all if it were deleted.